### DISCUSSION TITLE
Who can tell me How Running documentation locally

### DISCUSSION THREAD

**Discussion by anlaoliu** [twbs/bootstrap#33710] [Q&A] [answered]
``Run npm install to install the Node.js dependencies, including Hugo (the site builder).
Run npm run test (or a specific npm script) to rebuild distributed CSS and JavaScript files, as well as our docs assets.
From the root /bootstrap directory, run npm run docs-serve in the command line.
Open http://localhost:9001/ in your browser, and voilà.

The above is not detailed enough. Who can tell me the specific steps?
 In addition, I found that the content of the new site I created with Hugo is not consistent with the files in the site directory.

**Comment by florianlacreuse** [comment, marked as answer]
Hi,

Please follow this part of the doc first: https://getbootstrap.com/docs/5.0/getting-started/build-tools/#tooling-setup

Then you will be able to run `npm run docs-serve` within the `/bootstrap` directory as explained [here](https://getbootstrap.com/docs/5.0/getting-started/build-tools/#local-documentation).
